test/unit/test-amp-story-player.js

```diff
@@ -327,5 +335,39 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
 
       expect(playerEl.shadowRoot.querySelector('iframe')).to.exist;
     });
+
+    it('show callback builds corresponding adjacent iframes', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      await player.show('https://example.com/story3.html');
+
+      const stories = toArray(playerEl.querySelectorAll('a'));
+
+      expect(stories[0][IFRAME_IDX]).to.eql(undefined);
+      expect(stories[1][IFRAME_IDX]).to.eql(undefined);
+      expect(stories[2][IFRAME_IDX]).to.eql(0);
+      expect(stories[3][IFRAME_IDX]).to.eql(1);
+      expect(stories[4][IFRAME_IDX]).to.eql(2);
+    });
+
+    it('show callback should throw when story is not found', async () => {
```

Comment:
Can you add the same TODO you have to remove this test when we start automatically adding the story instead of throwing?